src/ComponentTester.cpp

```diff
@@ -30,18 +30,16 @@ void ComponentTester::testPin(){
   if(componentType.equalsIgnoreCase("BUZZER")){
     tone(pin, 1000, 100); //(Buzzer)_component.tone(1000) wouldn't work. Polymorphism?
   }
+
   if(componentType.equalsIgnoreCase("PHOTORESISTOR")){
-    for(int i = 1; i <= 10; i++){
-        if(Properties::DEBUGGING_ACTIVE) {
-          Logger::Log(String("Test #") + i + String(": ") + String(analogRead(pin)));
-        }
-      }
+        Serial.write(14022); //14022 Log, Info, Base Photoresistor Test
   }
+
   if(componentType.equalsIgnoreCase("BUTTON")){
     Button button(_component->getPin()); //May be wasted memory
     for(int i = 0; i < 150; i++){
       if(button.isPressed()) {
-        Logger::Log("The button was pressed");
+        Serial.write(24017); //24017 Data, Info, Button Pressed
```

Comment:
This should be a logging statement OR the `Log Level` should be `0`. Also - it should be `025` for `Button Pressed` - not `017` which is for `Alarm Failed`.